apis/openstack-cinder/src/main/java/org/jclouds/openstack/cinder/v2/config/CinderParserModule.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.openstack.cinder.v2.config;
+
+import com.google.inject.AbstractModule;
+import org.jclouds.json.config.GsonModule;
+import org.jclouds.json.config.GsonModule.DateAdapter;
+
+public class CinderParserModule extends AbstractModule {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/19701852/16475264/c74ba1da-3e9a-11e6-8c32-b2adffde15d1.png)

Rackspace-US andUK API's uses CinderParserModule. if we wants to keep it to common place we need to change in their code as well.